### PR TITLE
[IN-2117] disable homebrew install cleanup

### PR DESCRIPTION
**Pull request by Kanan-Bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Review by lmesz-bitrise — DISMISSED** [review]

**Review by MrBlaise — APPROVED** [review]

**Review by bitrise-tom — CHANGES_REQUESTED** [review]
As part of this PR, shouldn't we be running the cleanup as part of the build, then disabling on the end VMs?

**Comment by Kanan-Bitrise** [comment]
I wasn't aware of that, thanks!

File: roles/profiles/files/bashrc [hunk 1]
```diff
@@ -8,6 +8,8 @@ export LANGUAGE="en_US.UTF-8"
 #  Don't do auto update
 export HOMEBREW_NO_AUTO_UPDATE=1
 export HOMEBREW_NO_ANALYTICS=1
+# Don't do install cleanup
```
Comment (lmesz-bitrise) [inline roles/profiles/files/bashrc:11]: I would rephrase it to something like "Do not do execute brew cleanup during install" or simply remove it cause the variable talks itself

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## UPCOMING
 
+## `v2020_11_03_1`
+* `disable homebrew install cleanup`
```
Comment (lmesz-bitrise) [inline CHANGELOG.md:4]: I would rephrase it also

File: roles/weekly-update-shared/tasks/main.yml [hunk 3]
```diff
@@ -102,10 +102,16 @@
...
     path: "$HOME/{{ item.fold_path }}"
     state: absent
   with_items:
     - { fold_path: 'Library/Developer/Xcode/DerivedData' }
     - { fold_path: 'Library/Caches/com.apple.dt.Xcode/Downloads' }
+
+- name: Brew cleanup weekly
+  include_role:
+    name: brew-cleanup-weekly
+    tasks_from: main
```
Comment (lmesz-bitrise) [inline roles/weekly-update-shared/tasks/main.yml:117]: This is the default don't need to add it.

File: roles/brew-cleanup-weekly/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+---
+- name: Brew cleanup weekly
```
Comment (lmesz-bitrise) [inline roles/brew-cleanup-weekly/tasks/main.yml:2]: Hm... are you sure it will work like this? I think brew is installed under `/usr/local/bin` which is not part of the PATH

**Review by lmesz-bitrise — CHANGES_REQUESTED** [review]

**Comment by lmesz-bitrise** [comment]
Why did you just resolve the comments without any reaction?

**Comment by Kanan-Bitrise** [comment]
oh sorry forgot to push

File: roles/profiles/files/bashrc [hunk 5]
```diff
@@ -9,6 +9,8 @@ export LANGUAGE="en_US.UTF-8"
 export HOMEBREW_NO_AUTO_UPDATE=1
 export HOMEBREW_NO_ANALYTICS=1
 
+export HOMEBREW_NO_INSTALL_CLEANUP=1
```
Comment (bitrise-tom) [inline roles/profiles/files/bashrc:12]: remove the space above this and keep the env vars for homebrew together

File: roles/profiles/files/zshrc [hunk 6]
```diff
@@ -8,6 +8,8 @@ export LANGUAGE="en_US.UTF-8"
 export HOMEBREW_NO_AUTO_UPDATE=1
 export HOMEBREW_NO_ANALYTICS=1
 
+export HOMEBREW_NO_INSTALL_CLEANUP=1
```
Comment (bitrise-tom) [inline roles/profiles/files/zshrc:11]: unnecessary space

**Review by bitrise-tom — CHANGES_REQUESTED** [review]

**Review by bitrise-tom — APPROVED** [review]

**Review by lmesz-bitrise — APPROVED** [review]
